src/components/Chat/Chat.tsx

```diff
@@ -170,44 +179,39 @@ export const Chat = memo(() => {
     [dispatch],
   );
 
-  useEffect(() => {
-    if (!autoScrollEnabled || !chatContainerRef.current) {
-      return;
-    }
+  const setAutoScroll = () => {
```

Comment:
useCallback?

Reply:
@mikitabut , `setAutoScroll` isn't used in dependencies and doesn't use any changed or state properties(`disableAutoScrollTimeoutRef` always the same). That's why it isn't suggested to be converted to `useCallback`

Reply:
Got it
My undersatnding of useCallback was not right

Reply:
With `useCallback` function memorize one instance of function, but function will be created each time plus array of dependencies as arguments for `useCallback` and then `useCallback` works as Singleton with dependencies i.e. we don't have any optimization